wordpress/wp_mysql_logs/mysql/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignore mysql error.log, but keep folder so container can create log.
+# TODO: What about logrotated files?
```

Comment:
Update for logrotated file. Can logrotate from inside containers! Fixes issue with permissions.